static/app/components/activity/note/header.tsx

```diff
@@ -22,56 +21,53 @@ function NoteHeader({authorName, user, onEdit, onDelete}: Props) {
   const canEdit = activeUser && (activeUser.isSuperuser || user?.id === activeUser.id);
 
   return (
-    <div>
+    <Container>
       <ActivityAuthor>{authorName}</ActivityAuthor>
       {canEdit && (
-        <EditorTools>
-          <Tooltip
-            title={t('You can edit this comment due to your superuser status')}
-            disabled={!activeUser.isSuperuser}
-          >
-            <Edit onClick={onEdit}>{t('Edit')}</Edit>
-          </Tooltip>
-          <Tooltip
-            title={t('You can delete this comment due to your superuser status')}
-            disabled={!activeUser.isSuperuser}
-          >
-            <LinkWithConfirmation
-              title={t('Remove')}
-              message={t('Are you sure you wish to delete this comment?')}
-              onConfirm={onDelete}
-            >
-              <Remove>{t('Remove')}</Remove>
-            </LinkWithConfirmation>
-          </Tooltip>
-        </EditorTools>
+        <DropdownMenu
+          offset={4}
+          size="sm"
+          triggerProps={{
+            size: 'xs',
+            showChevron: false,
+            borderless: true,
+            icon: <IconEllipsis size="xs" />,
+            'aria-label': t('Comment Actions'),
+          }}
+          items={[
+            {
+              key: 'edit',
+              label: t('Edit'),
+              onAction: onEdit,
+              tooltip: activeUser.isSuperuser
```

Comment:
Seems like we should only display this tooltip when it's not your own comment, but this was the existing logic I guess

Reply:
Yeah I can update that too